### PR TITLE
Feat/to checksum address

### PR DESCRIPTION
**Pull request by dhruv500**
to checksum address func added in string extension

### CODE REVIEW

File: web3swift/src/Extensions/KeccakExtensions.swift [hunk 1]
```diff
@@ -33,3 +33,20 @@ public extension Web3Extensions where Base == String {
...
+        var result = "0x"
+        for i in 0 ... lowerCaseAddress.count - 1 {
+            if let val = Int(String(keccaf[i]), radix: 16), val >= 8 {
+                result.append(arr[i].uppercased())
+            } else {
+                result.append(arr[i])
+            }
+        }
+        return result
+    }
```
Comment (DarthMike) [inline web3swift/src/Extensions/KeccakExtensions.swift:51]: This should be an method in `EthereumAddress`

Comment (dhruv500) [reply web3swift/src/Extensions/KeccakExtensions.swift]: Done

File: web3swift/src/Extensions/HexExtensions.swift [hunk 2]
```diff
@@ -70,28 +70,38 @@ public extension Web3Extensions where Base == String {
...
+
+public extension String {
+    func stripHexPrefix() -> String {
+        if hasPrefix("0x") {
+            let indexStart = index(startIndex, offsetBy: 2)
+            return String(self[indexStart...])
+        }
+        return self
+    }
+}
```
Comment (DarthMike) [inline web3swift/src/Extensions/HexExtensions.swift:107]: We already have an extension on string `noHexPrefix`, why did you add this one here?

Comment (dhruvTor) [reply web3swift/src/Extensions/HexExtensions.swift]: my bad,removed.

**Review by DarthMike — APPROVED** [review]
Thanks @dhruv500
